Cargo.toml

```diff
@@ -16,6 +16,7 @@ digest = "0.10.6"
 either = "1.6.1"
 futures = "0.3.17"
 hex = "0.4.3"
+libc = "0.2.144"
```

Comment:
could you make this an optional dependency that goes along with memmap? Since we don't use libc unless the mmap feature is enabled.

Reply:
sure, done